src/Compilers/Shared/BuildServerConnection.cs

```diff
@@ -45,7 +45,7 @@ internal struct BuildPathsAlt
         /// </summary>
         internal string TempDirectory { get; }
 
-        internal BuildPathsAlt(string clientDir, string workingDir, string sdkDir, string tempDir)
+        internal BuildPathsAlt (string clientDir, string workingDir, string sdkDir, string tempDir)
```

Comment:
No space between method name and opening parens. Looked like the file has this in a lot of places.

Reply:
Yeah when I applied the .editorconfig it looks like it inserted them. Not really sure what happened there, I'm going to try to find another way to mechanically enforce the rules.